test/inductor/test_cudagraph_trees.py

```diff
@@ -44,6 +45,11 @@
     unittest.skipIf, not HAS_MULTIGPU, "requires multiple cuda devices"
 )
 
+if not TEST_CUDA_GRAPH:
+    if __name__ == "__main__":
+        sys.exit(0)
```

Comment:
This feels a bit out of place, please add it to https://github.com/pytorch/pytorch/blob/9567aaebe5385c2ce453c10c34180bba53773a93/test/inductor/test_cudagraph_trees.py#L1126-L1130

Reply:
I was following the syntax just a few lines above it 😅 . I've moved those to the bottom.